xl2times/transforms.py

```diff
@@ -13,6 +14,12 @@
 
 from tqdm import tqdm
 
+import logging
+import logging.config
+
+logger = logging.getLogger(__name__)
```

Comment:
Does this mean a run of the tool would result in two files `xl2times.log` and `transforms.log`?

Reply:
Ah that's a mistake actually, should be using just `from loguru import logger` here as its not an entrypoint.

But reasonable question - atm it'll create a log file for each separate entrypoint ( e.g. `run_benchmarks.log` and `xl2times.log`). I'm open to suggestions if you'd like different behaviour though?

Reply:
That sounds fine to me!